src/baldr/location.cc

```diff
@@ -26,8 +28,8 @@ bool Location::operator==(const Location& o) const {
          country_ == o.country_ && date_time_ == o.date_time_ && heading_ == o.heading_ &&
          heading_tolerance_ == o.heading_tolerance_ &&
          node_snap_tolerance_ == o.node_snap_tolerance_ && way_id_ == o.way_id_ &&
-         minimum_reachability_ == o.minimum_reachability_ && radius_ == o.radius_ &&
-         preferred_side_ == o.preferred_side_;
+         min_outbound_reach_ == o.min_outbound_reach_ && min_inbound_reach_ == o.min_inbound_reach_ &&
+         radius_ == o.radius_ && preferred_side_ == o.preferred_side_;
```

Comment:
lots of changes like this in here where we switch from single reach measure to directional reach measure